src/main/kotlin/com/sourcegraph/cody/commands/ui/CommandsTabPanel.kt

```diff
@@ -35,6 +45,16 @@ class CommandsTabPanel(
       addInlineEditActionButton("cody.documentCodeAction")
       addInlineEditActionButton("cody.testCodeAction")
     }
+
+    addHierarchyListener {
+      if (!project.isDisposed) {
+        if (it.component.isShowing) {
```

Comment:
Any reason we need to keep adding/removing it and cannot just leave it be?

Reply:
Happy to change this is a follow up... what would the trigger for removing the listener be?